### PR TITLE
Upgrade to powsybl-network-store.version 1.5.1

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

**Review by AbdelHedhili — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -108,6 +110,12 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-network-store-client</artifactId>
+            <version>${powsybl-network-store.version}</version>
```
Comment (antoinebhs) [inline pom.xml:113]: ?

File: pom.xml [hunk 2]
```diff
@@ -108,6 +110,12 @@
...
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-network-store-iidm-impl</artifactId>
+            <version>${powsybl-network-store.version}</version>
```
Comment (antoinebhs) [inline pom.xml:118]: `              
             <dependency>
                  <groupId>com.powsybl</groupId>
                  <artifactId>powsybl-network-store-model</artifactId>
                  <version>${powsybl-network-store.version}</version>
              </dependency>` ?

File: pom.xml (hunk 1 above)
Comment (etiennehomer) [reply pom.xml:114]: Done

File: pom.xml (hunk 2 above)
Comment (etiennehomer) [reply pom.xml:120]: Done

**Review by antoinebhs — APPROVED** [review]

**Pull request merged**